kube/resources/volume-workspace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: airbyte-volume-workspace
+  labels:
+    airbyte: volume-local
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 500Mi
```

Comment:
how do we handle deleting old jobs history and stuff? probably a not now problem but would be good to create the issue for it for when we break this due to monotonically increasing disk usage.

Reply:
https://github.com/airbytehq/airbyte/issues/1355